tools/internal_ci/linux/grpc_distribtests_python.sh

```diff
@@ -23,10 +23,19 @@ cd $(dirname $0)/../../..
 
 source tools/internal_ci/helper_scripts/prepare_build_linux_rc
 
-# some distribtests use a pre-registered binfmt_misc hook
-# to automatically execute foreign binaries (such as aarch64)
-# under qemu emulator.
-source tools/internal_ci/helper_scripts/prepare_qemu_rc
+IS_AARCH64_MUSL=""
+if [[ "${TASK_RUNNER_EXTRA_FILTERS}" == "aarch64 musllinux_1_1" || "${TASK_RUNNER_EXTRA_FILTERS}" == "presubmit aarch64 musllinux_1_1" ]]; then
+  IS_AARCH64_MUSL="True"
+fi
+
+if [[ "${IS_AARCH64_MUSL}" == "True" ]]; then
```

Comment:
Shouldn't we skip on aarch64 regardless of whether we're building musl artifacts?

Reply:
Not really, we're still crosscompiling some aarch64 artifacts, we don't need skip for them.